rocksdb_replicator/replicated_db.cpp

```diff
@@ -60,6 +60,8 @@ DEFINE_string(replicator_zk_cluster, "", "Zookeeper cluster");
 DEFINE_string(replicator_helix_cluster, "", "Helix cluster");
 DEFINE_int32(replication_error_reset_upstream_percentage, 10,
              "what percentage of replication errors should query helix for the latest leader");
+DEFINE_bool(reset_upstream_on_exception, false, 
```

Comment:
can you rename this to `reset_upstream_on_std_exception` since we always reset for replicate exceptions?